Directory.Packages.props

```diff
@@ -72,6 +72,7 @@
     <PackageVersion Include="Microsoft.VisualStudio.ProjectSystem.Managed" Version="17.2.0-beta1-20502-01" />
     <PackageVersion Include="Microsoft.VisualStudio.ProjectSystem.Managed.VS" Version="17.2.0-beta1-20502-01" />
     <PackageVersion Include="Microsoft.VisualStudio.ProjectSystem.VS" Version="17.4.221-pre" />
+    <PackageVersion Include="Microsoft.VisualStudio.SolutionPersistence" Version="1.0.28" />
```

Comment:
I know we often have to do something about source built packages when we introduce them. 

Something to add in: https://github.com/NuGet/NuGet.Client/blob/dev/eng/SourceBuildPrebuiltBaseline.xml.

Reply:
I'm not sure how much that will be relevant here, as this version of the package is already part of source build due to MSBuild and the SDK's reliance on it.

Reply:
I thought we usually need to include it in the repo list as long as it's being used by that repo. 

@mthalman, @MichaelSimons do we need it in the https://github.com/NuGet/NuGet.Client/blob/dev/eng/SourceBuildPrebuiltBaseline.xml?

Reply:
Yes, SolutionPersistence is already built in source build via https://github.com/dotnet/source-build-externals but NuGet doesn't consume SBE so it won't help with the resolving the prebuilt in this build. Adding it to the baseline is the right thing to do.

Reply:
I believe that would require NuGet to onboard to Arcade.